docs/permissions/data.md

```diff
@@ -88,13 +101,13 @@ See [impersonated view data permissions](./impersonation.md)
 
 {% include plans-blockquote.html feature="Blocked view data permission" %}
 
-**Blocked** ensures people in a group can’t see the data from this database, regardless of their permissions at the collection level.
+**Blocked** ensures people in a group can’t see the data from this database, schema, or table, regardless of their permissions at the collection level.
```

Comment:
One important thing to note is that Blocked prevents access to any native query in the database (even if set at the table level).

If a user is blocked on a table, they can't view any native query on that db